src/profile.ts

```diff
@@ -326,6 +328,7 @@ export default class Profile {
     this.bottomFillPattern = SolidFillPattern.MONOTONIC;
     this.bridgeFanSpeed = new Array(extruderCount).fill(100);
     this.cooling = new Array(extruderCount).fill(true);
+    this.coolingModuleSpeed = new Array(extruderCount).fill(100);
```

Comment:
These should be defined further down in the same order as the class member definitions above